### PR TITLE
Wasm command

### PR DESCRIPTION
**Pull request by philass**
My initial commits for the WASM backend

### CODE REVIEW

**Comment by athas** [comment]
I gave you push access to the repository.  You don't need to make PRs to push to your own (non-`master`) branches.

**Comment by philass** [comment]
Sorry I wasn't sure how to get my changes that exist in my fork into a branch of this repo, without making a PR. My intention was to only make one (this one).

File: src/Futhark/Actions.hs [hunk 1]
```diff
@@ -161,6 +191,28 @@ compileCAction fcfg mode outpath =
           liftIO $ writeFile cpath $ SequentialC.asExecutable cprog
           runCC cpath outpath ["-O3", "-std=c99"] ["-lm"]
 
+compileCtoWASMAction :: FutharkConfig -> CompilerMode -> FilePath -> Action SeqMem
+compileCtoWASMAction fcfg mode outpath =
+  Action
+    { actionName = "Compile to to sequential C",
```
Comment (melsman) [inline src/Futhark/Actions.hs:197]: to to -> to

Comment (philass) [reply src/Futhark/Actions.hs:197]: Good catch!

**Pull request merged**